frontend/src/lib/types.ts

```diff
@@ -0,0 +1,57 @@
+import subjectDetails from "../data/subjectDetails.json";
+import schedule from "../data/schedule.json";
+import emptySchedule from "../data/empty_schedule.json";
+
+export type Schedule = typeof schedule;
+
+export type SubjectDetails = typeof subjectDetails;
+
+export type EmptySchedule = typeof emptySchedule;
+
+export type Complex = "Any" | "NC" | "NR" | "Vikramshila";
```

Comment:
Why do we need `"Any"` type?

Reply:
"Any" is the intial value that we use for complex.